scripts/test-runner-filter.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+try {
+  const minimatch = require('minimatch');
+  const childProcess = require('child_process');
+  const gitDiffRange = (
+    process.env.TRAVIS_COMMIT_RANGE || 'master..HEAD'
+  ).replace('...', '..');
+  const changedFiles = childProcess
+    .execSync(`git diff --name-only ${gitDiffRange}`, {encoding: 'utf8'})
+    .split('\n')
+    .slice(0, -1);
+
+  const testRelatedFilePatterns = [
```

Comment:
How come there are files in this block that arent in the AppVeyor block?
Also would we not need eslint config here?

Reply:
Currently Appveyor only runs node tests, so all browser test related files do not need to trigger an appveyor build